app/index.js

```diff
@@ -69,7 +83,8 @@ function emitInitiateChange() {
       }
       logger.info(`${account} sending emitInitiateChange transaction`)
       let nonce = await getNonce()
-      consensus.methods.emitInitiateChange().send({ from: account, gas: process.env.GAS || 1000000, gasPrice: process.env.GAS_PRICE || '0', nonce: nonce })
+      let gasPrice = await getGasPrice()
+      consensus.methods.emitInitiateChange().send({ from: account, gas: process.env.GAS || 1000000, gasPrice: process.env.GAS_PRICE || gasPrice, nonce: nonce })
```

Comment:
can we define a minGasPrice in the config instead?

Reply:
so we will use like Math.max(process.env.MIN_GAS_PRICE, await getGasPrice())

Reply:
Can do but this should already be being done at the node level (using the changes I made to NM)

Reply:
perhaps add an extra buffer to the gas price in case the estimation that was returned is not high enough?

consensus.methods.emitInitiateChange().send({ from: account, gas: process.env.GAS || 1000000, gasPrice: process.env.GAS_PRICE || gasPrice * 1.5, nonce: nonce })